quickstart/config.yaml

```diff
@@ -10,8 +10,20 @@ sink:
   kafkaConnect:
     enabled: true
     url: http://kafka-connect:8083
-    clickhouse:
-      hostname: clickhouse
+    connectors:
```

Comment:
I think we should add it to `config.example.yaml` too

Reply:
I wanna change that config a bit anyway, so I'll do that in a followup PR.